### PR TITLE
MultiSlider fixes

### PR DESCRIPTION
**Pull request by jscheiny**
Several fixes to MultiSlider before final merge

### CODE REVIEW

File: packages/core/src/components/slider/slider.tsx [hunk 1]
```diff
@@ -42,24 +41,18 @@ export class Slider extends AbstractPureComponent<ISliderProps> {
...
+        const { initialValue, value, onChange, onRelease, ...props } = this.props;
         return (
-            <MultiSlider {...props} onChange={this.handleChange} onRelease={this.handleRelease}>
+            <MultiSlider {...props}>
                 <MultiSlider.Handle
                     value={value}
                     intentAfter={value < initialValue ? Intent.PRIMARY : undefined}
                     intentBefore={value >= initialValue ? Intent.PRIMARY : undefined}
+                    onChange={onChange}
+                    onRelease={onRelease}
```
Comment (giladgray) [inline packages/core/src/components/slider/slider.tsx]: 👌

File: packages/core/src/components/slider/multiSlider.tsx [hunk 2]
```diff
@@ -369,9 +369,21 @@ export class MultiSlider extends AbstractPureComponent<IMultiSliderProps, ISlide
         const oldValues = this.handleProps.map(handle => handle.value);
         if (!Utils.arraysEqual(newValues, oldValues)) {
             Utils.safeInvoke(this.props.onChange, newValues);
+            this.handleProps.forEach((handle, index) => {
+                if (oldValues[index] !== newValues[index]) {
```
Comment (giladgray) [inline packages/core/src/components/slider/multiSlider.tsx]: 👌

File: packages/core/src/components/slider/handleProps.tsx [hunk 3]
```diff
@@ -37,9 +37,6 @@ export interface IHandleProps extends IProps {
     /** Numeric value of this handle. */
     value: number;
 
-    /** Whether this handle is non-interactive. */
-    disabled?: boolean;
```
Comment (giladgray) [inline packages/core/src/components/slider/handleProps.tsx]: add this to `IHandleInternalProps` then? pretty sure it is respected in `<Handle>`

Comment (jscheiny) [reply packages/core/src/components/slider/handleProps.tsx]: Yeah, fixed.

Comment (giladgray) [reply packages/core/src/components/slider/handleProps.tsx]: tho it would be easy enough to support this prop, like you did with `onChange/onRelease`

**Pull request merged**